layouts/account.tsx

```diff
@@ -41,13 +45,39 @@ type TabTypeEnum = "delegating" | "orchestrating" | "history";
 
 const ACCOUNT_VIEWS: TabTypeEnum[] = ["delegating", "orchestrating", "history"];
 
-const AccountLayout = ({
-  account,
-  sortedOrchestrators,
-}: {
-  account?: AccountQueryResult["data"] | null;
-  sortedOrchestrators: OrchestratorsSortedQueryResult["data"];
-}) => {
+const AccountLayout = () => {
+  /* PART OF https://github.com/livepeer/explorer/pull/427 - REMOVE ONCE SERVER-SIDE ISSUE IS FIXED */
+  const context = { params: useParams() };
+  const [sortedOrchestrators, setSortedOrchestrators] = useState<
+    OrchestratorsSortedQueryResult["data"] | null
+  >(null);
+  const [account, setAccount] = useState<AccountQueryResult["data"] | null>(
+    null
+  );
+
+  useEffect(() => {
```

Comment:
Normally I'd use something like `useSWR` or react-query's `useQuery` as those make it a bit more readable, and provide persistence/refresh on refocus. Might not be worth it as it's a temporary fix, up to you.